client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
 			post_id: 32011,
-			title: 'The Reader: An Overview',
-			description:
+			title: translate( 'The Reader: An Overview' ),
+			description: translate(
 				'Read posts from all the sites you follow — even ones that aren’t on WordPress.com! ' +
-				'Discover great new reads and keep track of your comments and replies in one convenient place.',
+					'Discover great new reads and keep track of your comments and replies in one convenient place.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/following/',
-			title: 'Follow Blogs',
-			description:
-				'When you follow a blog on WordPress.com, new posts from that site will automatically appear in your Reader.',
+			title: translate( 'Follow Blogs' ),
+			description: translate(
+				'When you follow a blog on WordPress.com, new posts from that site will automatically appear in your Reader.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/topics/',
 			post_id: 2166,
-			title: 'Following Specific Topics in the Reader',
-			description:
+			title: translate( 'Following Specific Topics in the Reader' ),
+			description: translate(
 				'Looking for posts on a specific topic? Besides following entire blogs, you can also follow posts on a specific subject ' +
-				'from across WordPress.com. You do this by adding the topic you’re interested in under the Tags heading in the Reader.',
+					'from across WordPress.com. You do this by adding the topic you’re interested in under the Tags heading in the Reader.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/grow-your-community/',
-			title: 'Grow Your Community',
-			description:
-				'You’ve worked hard on building your site, now it’s time to explore the community and get noticed.',
+			title: translate( 'Grow Your Community' ),
+			description: translate(
+				'You’ve worked hard on building your site, now it’s time to explore the community and get noticed.'
+			),
 		},
 	],
 	help: [
 		{
 			link: 'http://en.support.wordpress.com/blogging-u/',
 			post_id: 117437,
-			title: 'Blogging U.',
-			description:
+			title: translate( 'Blogging U.' ),
+			description: translate(
 				'Blogging U. courses deliver free expert advice, pro tips, and inspiration right to your ' +
-				'email inbox. Sign up now!',
+					'email inbox. Sign up now!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/help-support-options/',
 			post_id: 149,
-			title: 'Help! Getting WordPress.com Support',
-			description:
-				'WordPress.com offers a number of avenues for reaching helpful, individualized support.',
+			title: translate( 'Help! Getting WordPress.com Support' ),
+			description: translate(
+				'WordPress.com offers a number of avenues for reaching helpful, individualized support.'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/',
-			title: 'All Support Articles',
-			description: 'Looking to learn more about a feature? Our docs have all the details.',
+			title: translate( 'All Support Articles' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'All support articles' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).